.github/workflows/nv-sd.yml

```diff
@@ -55,6 +55,7 @@ jobs:
       - name: Unit tests
         run: |
           unset TORCH_CUDA_ARCH_LIST # only jit compile for current arch
+          python download_sd_model.py
```

Comment:
I'd probably move this to tests/unit/utils or test/unit/inference?

Reply:
Script was removed, marking as resolved.